### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by cathysull**
After this commit, I'll add an example for us to add latest additions, fixes, etc. for the not yet released next package.

### CODE REVIEW

**Review by sunandabalu — APPROVED** [review]
LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changelog
+All notable changes to the .NET Upgrade Assistant will be documented in this file. Version numbers below will follow (best effort) the corresponding NuGet package versions here: https://www.nuget.org/packages/upgrade-assistant/
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+
+## Version 0.2.212405 - Preview 1 - 2021-02-24  ([Link](https://www.nuget.org/packages/upgrade-assistant/0.2.212405))
+
+The Preview 1 release, was the first public release that supports upgrading ASP.NET MVC, Windows Forms, WPF, Console, and Class Libraries .NET Framework applications to .NET 5.
```
Comment (twsouthwick) [inline CHANGELOG.md:9]: I tend to like an `Added`, `Fixed`, etc to help organize changes. Probably doesn't make sense for the first release, though

**Review by twsouthwick — APPROVED** [review]

**Review by mjrousos — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (cathysull) [reply CHANGELOG.md:9]: Yes, of course! I was going to add that in the releases going forward.

**Pull request merged**